mmdet/core/bbox/assigners/approx_max_iou_assigner.py

```diff
@@ -119,7 +119,7 @@ def assign(self,
         bboxes = squares[:, :4]
 
         if (self.ignore_iof_thr > 0) and (gt_bboxes_ignore is not None) and (
-                gt_bboxes_ignore.numel() > 0):
+                gt_bboxes_ignore.numel() > 0 and bboxes.numel() > 0):
```

Comment:
Thanks for the fix! Since there are lots of conditions, we can just use one pair of brackets to hold them.
```python
if (self.ignore_iof_thr > 0 and gt_bboxes_ignore is not None
    and gt_bboxes_ignore.numel() > 0 and bboxes.numel() > 0):
    pass
```